src/Parser/DemonHunter/Havoc/Modules/Items/DelusionsOfGrandeur.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+import SpellLink from 'common/SpellLink';
+import ItemLink from 'common/ItemLink';
+import { formatNumber, formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+import Wrapper from 'common/Wrapper';
+import FuryTracker from '../ResourceTracker/FuryTracker';
+
+/*
+* Equip: The remaining cooldown on Metamorphosis is reduced by 1 sec for every 30 Fury you spend.
+*/
+
+class DelusionsOfGrandeur extends Analyzer {
+	static dependencies = {
+		combatants: Combatants,
+		SpellUsable: SpellUsable,
+		furyTracker: FuryTracker,
+	}
+
+	on_initialized() {
+		this.active = this.combatants.selected.hasShoulder(ITEMS.DELUSIONS_OF_GRANDEUR.id);
+	}
+
+	get percentCDRWasted(){
+		return this.furyTracker.cooldownReductionWasted / this.furyTracker.cooldownReduction;
+	}
+
+	get suggestionThresholds() {
+    return {
+      actual: this.percentCDRWasted,
+      isGreaterThan: {
+        minor: 0.05,
+        average: 0.15,
+        major: 0.25,
+      },
+      style: 'percentage',
+    };
+  }
+
+  suggestions(when) {
+  	when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) => {
+  		return suggest(<Wrapper>You wasted {formatPercentage(actual)}% of the cooldown reduction from <ItemLink id={ITEMS.DELUSIONS_OF_GRANDEUR.id} icon/>. Make sure to be using <SpellLink id={SPELLS.METAMORPHOSIS_HAVOC.id} icon/> on cooldown with this item.</Wrapper>)
+  			.icon(ITEMS.DELUSIONS_OF_GRANDEUR.icon)
+  			.actual(`${formatNumber(this.furyTracker.cooldownReductionWasted)} seconds wasted`)
+  			.recommended(`Wasting less than ${formatPercentage(recommended)}% is recommended.`);
+  	});
+  }
+
+	item() {
+		return {
+			item: ITEMS.DELUSIONS_OF_GRANDEUR,
+			result:(
+				<dfn data-tip={`You wasted ${formatNumber(this.furyTracker.cooldownReductionWasted)} second of cooldown reduction.`}>
+					<Wrapper>Reduced the cooldown of <SpellLink id={SPELLS.METAMORPHOSIS_HAVOC.id} icon/> by {formatNumber(this.furyTracker.cooldownReduction / this.owner.fightDuration * 60000)} seconds per minute.</Wrapper>
```

Comment:
I kinda think 'seconds per minute' is an awkward metric, 'seconds per cast' makes more sense to me. Let me know what you think about that.

Reply:
Per cast of the ability? What about reduced the cool down from X to Y. I was honestly unsure about what to display here

Reply:
You could show it the way CDR is shown [for Brewmasters](https://wowanalyzer.com/report/nxzcqJKmYwHbdWMp/8-Heroic+Felhounds+of+Sargeras+-+Kill+(4:02)/Eisenpelz). The calculation for that [is here](https://github.com/WoWAnalyzer/WoWAnalyzer/blob/master/src/Parser/Monk/Brewmaster/Modules/Core/BrewCDR.js#L48-L50).

This number is (in aggregate) the same as CDR in league of legends (so 50% CDR means your cooldowns are effectively cut in half over the course of the fight, 100% means no cooldown (but is unreachable fwiw)).

Reply:
Yeah, % CDR is reasonable display, and so is "abilities average effective cooldown was X (reduced from Y)"